### DISCUSSION TITLE
Accessing COGs from Azure Blob Storage

### DISCUSSION THREAD

**Discussion by aliasmrchips** [developmentseed/titiler#264] [Q&A] [answered]
Is there any particular reason that urls of the form `az://...` don't work, while `/vsiaz/...` works just fine? For example, using the cog/viewer endpoint?

**Comment by vincentsarago** [comment, marked as answer]
I'm pretty sure you are hitting this https://github.com/mapbox/rasterio/pull/1906#issuecomment-780661937

`az://` was introduced in Rasterio 1.2 so if you are using Rasterio 1.1.8 you have to use either the https://blob or /vsiaz form. For `private` object, you'll need to have Rasterio 1.2.1 

Note:
We are recommending to use Rasterio 1.1.8 👇 

https://github.com/developmentseed/titiler/blob/b5bcb0f0acc6a250697022f900d730d5e8558acd/Dockerfile#L9-L11

**Comment by aliasmrchips** [comment]
Ah! I saw the warning, so I should have taken that into account when I investigated.
